.SRCINFO

```diff
@@ -1,26 +1,26 @@
 pkgbase = jenkins-lts
 	pkgdesc = Extendable continuous integration server (lts)
-	pkgver = 2.138.2
+	pkgver = 2.138.3
 	pkgrel = 1
 	url = https://jenkins.io
 	install = jenkins.install
 	arch = any
 	license = MIT
-	depends = java-runtime-headless>=8
+	depends = jre8-openjdk-headless
```

Comment:
I agree that we might limit jenkins to version 8 due the limited support. But I disagree depending directly on `jre8-openjdk-headless` since it excludes several environment providing jre8 functionality.

I'd suggest to change the line to:

```
depends = java-runtime-headless=8
```